cas-server-support-osf/src/main/java/org/pac4j/oauth/client/OrcidClient.java

```diff
@@ -108,7 +107,7 @@ protected boolean hasBeenCancelled(final WebContext context) {
     @Override
     protected String getProfileUrl(final Token accessToken) {
         if (accessToken instanceof OrcidToken) {
-            return String.format("https://%s.orcid.org/v1.2/%s/orcid-profile",
+            return String.format("https://%s.orcid.org/v2.0/%s/record",
```

Comment:
Profile endpoint is changed: https://github.com/ORCID/ORCID-Source/tree/master/orcid-model/src/main/resources/record_2.0#calls